x/oracle/keeper/params.go

```diff
@@ -63,6 +63,32 @@ func (k Keeper) MinValidPerWindow(ctx sdk.Context) (res sdk.Dec) {
 	return
 }
 
+// StampPeriod returns the amount of blocks the historacle module waits
+// between recording a set of prices.
+func (k Keeper) StampPeriod(ctx sdk.Context) (res uint64) {
+	k.paramSpace.Get(ctx, types.KeyStampPeriod, &res)
+	return
+}
+
+// SetStampPeriod updates the amount of blocks the historacle module waits
```

Comment:
```suggestion
// SetStampPeriod updates the amount of blocks the historacle endblocker waits
```

Same here, technically historacle isn't a module of its own ^